client/container_list.go

```diff
@@ -2,9 +2,11 @@ package client
 
 import (
 	"encoding/json"
+	"errors"
 	"net/url"
 	"strconv"
 
+	"github.com/docker/docker/pkg/version"
```

Comment:
we cannot import a docker package, it's design constraint, sorry. We should probably extract that package from docker or use a different one.